GOVERNANCE.md

```diff
@@ -0,0 +1,130 @@
+# Release Working Group
+
+The Node.js Release Working Group (WG) maintains oversight
+over the Node.js Release and Long Term Support (LTS) teams. It
+manages the release and long term support shedule and policies
+for all Node.js releases.
+
+The WG has final authority over Releases including:
+
+* Release process and tools.
+* Content for all releases.
```

Comment:
Yep, generally we've always operated on the model that the person actually doing the release has the final say about what is included in the release. The CTC may decide to include something, but if the releaser finds an issue that would block the release, they should have the freedom to omit it until the issue is resolved.